src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
The file checksum will be constant unless the certificate is updated. Of course, the ctime and mtime will change everytime it is rewritten, so that could trigger an audit. I agree with @NOYB that building an array and policing files that way is probably better than always deleting and recreating. Probably validating contents by checking file size is a reasonable, low-overhead way of checking that the file hasn't been accidentally truncated or overwritten. Obviously that's not security safe, but it does provide some reasonable sanity checking.